src/rust/ensogl/src/display/symbol/gpu/geometry/compound/sprite.rs

```diff
@@ -119,6 +119,18 @@ impl Size {
 }
 
 
+// === Getters ===
+
+#[allow(missing_docs)]
+impl Size {
+    #[deprecated(note = "This will be removed without replacement soon.")]
+    pub fn get(&self) -> Vector2<f32>{
+        if self.hidden.get() { self.value.get() }
+        else                 { self.attr.get()  }
+    }
+}
```

Comment:
Let's add more context here. Someone reading such things should understand WHY it is deprecated. We want to keep here `Var`s instead of normal values, so `get` will not result in value soon.